spec.html

```diff
@@ -32086,27 +32086,127 @@ <h1>Week Day</h1>
-        <p>If an implementation does not support a conversion described above or if political rules for time _t_ are not available within the implementation, the result must be *+0*<sub>𝔽</sub>.</p>
+        <p>
+          When the input represents a local time repeating multiple times at a negative time zone transition (e.g. when the daylight saving time ends or the time zone offset is decreased due to a time zone rule change), the returned List will have more than one element. The elements are sorted in numerical order.
+          When the input represents a skipped local time at a positive time zone transition (e.g. when the daylight saving time starts or the time zone offset is increased due to a time zone rule change), the returned List will be empty.
+          Otherwise, the returned List will have one element.
+        </p>
+        <p>The minimum implementation of GetIANATimeZoneEpochValue for ECMAScript implementations that do not include local political rules for any time zones performs the following steps when called:</p>
+        <emu-alg>
+          1. Assert: _timeZoneIdentifier_ is *"UTC"*.
+          1. Let _epochNanoseconds_ be GetEpochFromISOParts(_year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_).
+          1. Return &laquo; _epochNanoseconds_ &raquo;.
+        </emu-alg>
         <emu-note>
           <p>It is recommended that implementations use the time zone information of the IANA Time Zone Database <a href="https://www.iana.org/time-zones/">https://www.iana.org/time-zones/</a>.</p>
-          <p>1:30 AM on 5 November 2017 in America/New_York is repeated twice (fall backward), but it must be interpreted as 1:30 AM UTC-04 instead of 1:30 AM UTC-05. LocalTZA(TimeClip(MakeDate(MakeDay(2017, 10, 5), MakeTime(1, 30, 0, 0))), false) is <emu-eqn>-4 &times; msPerHour</emu-eqn>.</p>
-          <p>2:30 AM on 12 March 2017 in America/New_York does not exist, but it must be interpreted as 2:30 AM UTC-05 (equivalent to 3:30 AM UTC-04). LocalTZA(TimeClip(MakeDate(MakeDay(2017, 2, 12), MakeTime(2, 30, 0, 0))), false) is <emu-eqn>-5 &times; msPerHour</emu-eqn>.</p>
-          <p>Local time zone offset values may be positive <i>or</i> negative.</p>
+          <p>1:30 AM on 5 November 2017 in America/New_York is repeated twice (fall backward), so GetIANATimeZoneEpochValue(*"America/New_York"*, 2017, 11, 5, 1, 30, 0, 0, 0, 0) would return a List of length 2.</p>
+          <p>2:30 AM on 12 March 2017 in America/New_York does not exist, so GetIANATimeZoneEpochValue(*"America/New_York"*, 2017, 3, 12, 2, 30, 0, 0, 0, 0) would return an empty List.</p>
+        </emu-note>
+      </emu-clause>
+
+      <emu-clause id="sec-getianatimezoneoffsetnanoseconds" type="implementation-defined abstract operation">
+        <h1>
+          GetIANATimeZoneOffsetNanoseconds (
+            _epochNanoseconds_: a BigInt,
+            _timeZoneIdentifier_: a String,
+          ): an integer
```

Comment:
This is a UTC offset, and the others are nanoseconds since the epoch. There's no reason in this PR specifically for nanoseconds since the epoch to be expressed as a bigint, but preserving that distinction is in the interest of aligning with Temporal, since exact time is stored as a BigInt in Temporal.Instant and Temporal.ZonedDateTime.